src/main/java/org/takes/tk/TkText.java

```diff
@@ -65,6 +67,11 @@ public TkText(final String body) {
             new Take() {
                 @Override
                 public Response act(final Request req) {
+                    try {
+                        new RqPrint(req).printBody();
+                    } catch (final IOException exc) {
```

Comment:
@yegor256 IOException is a checked exception. Signature `public Response act(final Request req)` doesn't contain `throws IOException` so I cannot to let it float up